tests/test_concurrent_operations.rs

```diff
@@ -50,6 +50,17 @@ fn test_concurrent_operations_auto_rebase() {
     std::fs::write(repo_path.join("file"), "contents").unwrap();
     test_env.jj_cmd_success(&repo_path, &["describe", "-m", "initial"]);
     let stdout = test_env.jj_cmd_success(&repo_path, &["op", "log"]);
+    insta::assert_snapshot!(stdout, @r###"
+    @ cffade45feb4 test-username@host.example.com 2001-02-03 04:05:08.000 +07:00 - 2001-02-03 04:05:08.000 +07:00
+    | describe commit 123ed18e4c4c0d77428df41112bc02ffc83fb935
+    | args: /usr/local/google/home/martinvonz/jj/target/debug/jj describe -m initial
```

Comment:
Needs to override or drop the executable path.

Reply:
I meant override/drop the executable path stored in the op log, which would affect the op hash.

Reply:
Yeah, I saw the tests fail. I think I'll just replace it by "jj". I suppose it could be useful for troubleshooting purposes to also know the executable's path, but we can deal with that later if we need it.